README.md

```diff
@@ -16,7 +16,7 @@ Open API definition is considered as the single source of truth to the WSO2 API
 
 ![alt text](https://raw.githubusercontent.com/wso2/k8s-apim-operator/master/apim-operator.png)
 
-The developer first approach is used when creating the API Manager Operator for Kubernetes. When an user requires to expose an API for the service he created, he only needs to provide the Open API definition to the Kubernetes. Then it will create the API and deploy his API in the WSO2 API Microgateway. His API is exposed as the Load Balancer service type in Kubernetes. 
+The developer first approach is used when creating the API Manager Operator for Kubernetes. When a user requires to expose an API for the service he created, he only needs to provide the Open API definition to the API Manager command line tool. API Manager command line tool can take open API definition take open API as input argument and pass it to kaniko job running on kubernetes. This job will create docker image and push it to the remote registry. Then it will create API Microgateway runtime from that docker image and deploy it in kubernetes. This API will be exposed as Load Balancer service type in Kubernetes. 
```

Comment:
The developer first approach is used when creating the API Manager Operator for Kubernetes. When a user requires to expose an API for the service he created, he only needs to provide the Open API definition to the API Manager command-line tool. API Manager command-line tool can take open API definition as the input argument and pass it to a kaniko job running on Kubernetes. This job will create a docker image and push it to the remote registry. Then it will create API Microgateway runtime from that